### PR TITLE
Expand material and gesture links

### PR DESCRIPTION
**Pull request by sethladd**
Fixes https://github.com/flutter/flutter/issues/8369

### CODE REVIEW

**Comment by sethladd** [comment]
cc @Sfshaza does this help?

**Comment by Sfshaza** [comment]
LGTM!

**Comment by sethladd** [comment]
Thanks!

File: gestures.md [hunk 1]
```diff
@@ -87,12 +87,17 @@ lifecycle of the gesture (e.g., drag start, drag update, and drag end):
...
-to listen to taps, and a
+If you're using Material widgets, many of those widgets already respond
+to taps or gestures.
+For example,
+[IconButton](https://docs.flutter.io/flutter/material/IconButton-class.html) and
+[FlatButton](https://docs.flutter.io/flutter/material/FlatButton-class.html)
+respond to presses (taps), and
 [`ListView`](https://docs.flutter.io/flutter/widgets/ListView-class.html)
-to listen to swipes to trigger scrolling.
+responded to swipes to trigger scrolling.
```
Comment (Hixie) [inline gestures.md]: s/responded/responds/

Comment (sethladd) [inline gestures.md]: Doh! thanks

**Pull request merged**